Basic-DSA/GenericLinkedListImpl.java

```diff
@@ -0,0 +1,150 @@
+class Node<T> {
+    T data;
+    Node<T> next;
+    Node(T data) {
+        this.data = data;
+        this.next = null;
+    }
+}
+
+interface GenericList<T> {
+    public void addFirst(T data);
+    public void addLast(T data);
+    public void deleteFirst();
+    public void deleteLast();
+    public void printList();
+    public void addAtIndex(int index, T data);
+    public void deleteAtIndex(int index);
```

Comment:
add a method to return size
`public int size();`

Reply:
ok